### ISSUE TITLE
Playstore link not working

### ISSUE THREAD

**Issue by Azzurous** [Drutol/MALClient#292]
### TL;DR Version
Your app is missing from the Playstore
### Detailed Version
I was planning on installing the Malclient app on my new phone however i could not find it anywhere. Also i don't think it's a region locking issue since if that were the case the app would at least show up in the installed section of the play store when viewed on my old phone but it doesn't.

![image](https://user-images.githubusercontent.com/38433708/91242612-33be6a00-e748-11ea-8e00-ac08950ebff1.png)
( Result of click link on this Github page ^ )

And so i hope this issue can be solved quickly but i get the feeling that this might not be as easy a fix as i imagine, so perhaps a direct apk download could be provided

**Comment by Drutol** [comment]
Hello, google decided to ninja pull the app from the store because they did not like paypal link inside... working on getting it restored.

**Comment by Drutol** [comment]
Speaking of which it looks like they have restored the app just now ^^
